react/package.json

```diff
@@ -13,16 +13,15 @@
   },
   "devDependencies": {
     "@apollo/react-testing": "^3.1.3",
-    "@types/jest": "^25.1.2",
-    "@types/node": "^13.7.1",
-    "@types/react": "^16.9.19",
-    "@vtex/test-tools": "^3.0.3",
-    "@vtex/tsconfig": "^0.4.3",
+    "@types/jest": "^25.1.4",
+    "@types/node": "^13.9.8",
+    "@types/react": "^16.9.31",
+    "@vtex/test-tools": "^3.1.0",
+    "@vtex/tsconfig": "^0.4.4",
     "apollo-cache-inmemory": "^1.6.5",
-    "eslint-config-vtex-react": "^6.1.1",
-    "eslint": "^6.8.0",
     "graphql": "^14.6.0",
     "prop-types": "^15.7.2",
-    "typescript": "3.7.3"
+    "typescript": "3.8.3",
+    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.98.1/public/@types/vtex.render-runtime"
```

Comment:
Should this be here?

Reply:
`vtex setup` thingie 🤷‍♂

Reply:
I would remove it, since it doesnt work properly